mkdocs.yml

```diff
@@ -1,6 +1,6 @@
 site_name: 'Pi-hole documentation'
 site_url: 'https://docs.pi-hole.net/'
-repo_url: 'https://github.com/pi-hole/docs'
+repo_url: 'https://github.com/pi-hole/pi-hole'
```

Comment:
This isn't right. It will break with the edit button.

Reply:
Meaning?

Reply:
![Screenshot_2020-02-09 Overview of Pi-hole](https://user-images.githubusercontent.com/2782850/74129022-6da49600-4b93-11ea-96a6-087db575d323.png)

This is what is there now, and that is the wrong stars/forks. That needs to be the stars/forks for the core project, 20.6k Stars, 1.4k Forks.

Reply:
Is the edit URL correct for each page?

Reply:
If it comes down to having an edit link or the right number of stars/forks then we have the right number of stars/forks and no edit link.

Reply:
All I'm saying is that this patches introduces 404 links for the Edit URLs.

Reply:
Well, then something changed and introduced that in the last few days/weeks. We used to have the stars/forks for the core repo in that top area.

Reply:
I know what changed (I changed it).

I pushed a patch which should make the edit URL work again.